libs/domain/content/link/link.component.ts

```diff
@@ -1,19 +1,42 @@
 import { resolve } from '@spryker-oryx/di';
 import { ContentMixin } from '@spryker-oryx/experience';
 import { LinkService } from '@spryker-oryx/site';
-import { computed, hydrate } from '@spryker-oryx/utilities';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import {
+  computed,
+  elementEffect,
+  hydrate,
+  ssrShim,
+} from '@spryker-oryx/utilities';
 import { LitElement, TemplateResult, html } from 'lit';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { when } from 'lit/directives/when.js';
-import { ContentLinkContent, ContentLinkOptions } from './link.model';
+import {
+  ContentLinkAppearance,
+  ContentLinkContent,
+  ContentLinkOptions,
+} from './link.model';
+import { styles } from './link.styles';
 
+@ssrShim('style')
 @hydrate()
 export class ContentLinkComponent extends ContentMixin<
   ContentLinkOptions,
   ContentLinkContent
 >(LitElement) {
+  static styles = styles;
+
   protected semanticLinkService = resolve(LinkService);
 
+  @elementEffect()
+  protected setDropdown = () => {
+    const { appearance } = this.$options();
+    this.toggleAttribute(
+      'dropdown',
+      appearance === ContentLinkAppearance.DROPDOWN
```

Comment:
Maybe it makes sense to have an `appearance` as an attribute as long as we need it for styling? To not have this additional attribute.